documentation/package.json

```diff
@@ -82,7 +83,7 @@
     "webpack": "^5.75.0"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
+    "node": "16.* || >= 18"
```

Comment:
Updated this as our floated deps was failing due to `ember-url-hash-polyfill` requires node 16+ now and so does the lastest `rollup-ts`.